### PR TITLE
chore(bot): print detailed App studio error msg from response

### PR DESCRIPTION
**Pull request by yukun-dong**
before:
3ab4719ef4e49930a269235a9753c99
after:
image

### CODE REVIEW

File: packages/fx-core/src/plugins/resource/bot/errors.ts [hunk 1]
```diff
@@ -104,18 +105,33 @@ export class PluginError extends Error {
...
+    msg += this.getAppStudioErrorDetails(this.innerError);
     if (this.suggestions.length > 0) {
       msg += getDefaultString("plugins.bot.ErrorSuggestions", this.suggestions.join(" "));
     }
     return msg;
   }
+
   genDisplayMessage(): string {
     let msg = `${this.details[1]} `;
+    msg += this.getAppStudioErrorDetails(this.innerError);
```
Comment (eriolchan) [inline packages/fx-core/src/plugins/resource/bot/errors.ts:117]: Why don't we also output inner error besides AppStudio Error?

Comment (eriolchan) [reply packages/fx-core/src/plugins/resource/bot/errors.ts:117]: We can have a better structure of the error message. E.g.
message: ..., suggestion: ..., innerError: ...

Comment (yukun-dong) [reply packages/fx-core/src/plugins/resource/bot/errors.ts:117]: The `getAppStudioErrorDetails` actually returns the inner error message, I will change it to a better name.

File: packages/fx-core/src/plugins/resource/bot/errors.ts [hunk 2]
```diff
@@ -297,42 +298,50 @@ export function wrapError(
   sendTelemetry: boolean,
   name: string
 ): FxResult {
-  let errorMsg = isErrorWithMessage(e) ? e.message : "";
+  let detailedErrorMsg = isErrorWithMessage(e) ? e.message : "";
```
Comment (eriolchan) [inline packages/fx-core/src/plugins/resource/bot/errors.ts:301]: Looking through the code, it looks we can still use errorMsg variable.

**Review by eriolchan — APPROVED** [review]

**Review by IvanJobs — APPROVED** [review]
LGTM, Great Job!

**Pull request merged**